### PR TITLE
Handle all schemas in api/records/extent and region.getmap

### PR DESCRIPTION
**Pull request by jonescc**
image

### CODE REVIEW

File: services/src/main/java/org/fao/geonet/api/regions/metadata/MetadataRegionSearchRequest.java [hunk 1]
```diff
@@ -196,9 +200,25 @@ private MultiPolygon getMultiPolygon(HashSet<Polygon> polygons) {
     }
 
     Region parseRegion(Id mdId, Element extentObj) throws Exception {
```
Comment (fxprunayre) [inline services/src/main/java/org/fao/geonet/api/regions/metadata/MetadataRegionSearchRequest.java]: Should we remove that also which is schema dependent? Here it only works for ISO19139

Comment (jonescc) [reply services/src/main/java/org/fao/geonet/api/regions/metadata/MetadataRegionSearchRequest.java]: To do that, the extract-gml in all schema plugins would need to be able to handle xml fragments which was soemthing outside the scope of what I was looking at (the spatial extent for the metadata record as a whole).

Comment (fxprunayre) [reply services/src/main/java/org/fao/geonet/api/regions/metadata/MetadataRegionSearchRequest.java]: Sure. I was not sure about the usage of parseRegion (maybe we could simply remove it ?) I was also looking to Elasticsearch branch where extract-gml is basically removed (geometries are stored in the index and handled by the main XSLT for indexing).

Comment (jonescc) [reply services/src/main/java/org/fao/geonet/api/regions/metadata/MetadataRegionSearchRequest.java]: I'm treading carefully here, trying not to break (or remove) anything - only adding.  Not really sure who may or may not be using these apis.  I'm able to work out whether geonetwork core uses something or not, but not what third parties use.  If parseRegion is not used - yes would be good to remove.

Comment (fxprunayre) [reply services/src/main/java/org/fao/geonet/api/regions/metadata/MetadataRegionSearchRequest.java]: Sure. We can do clean up in elastic branch.

**Pull request merged**